yellow/false-positive-treatment.md

```diff
@@ -1,16 +1,24 @@
 # False Positive Treatment
 
-In order to not overlook real vulnerabilities, false positives are suppressed so they will not show up on the next tests again.
+Suppress false positives so they will not show up on the next tests again. This will help you to see the real vulnerabilities (not hidden among many many false positives anymore).
```

Comment:
```suggestion
Suppress false positives so that they will not show up on the next tests again. This will help you to see the real vulnerabilities (not hidden among many many false positives anymore).
```